admin_guide/managing_networking.adoc

```diff
@@ -1246,6 +1246,67 @@ IP address. If `egressIPs` is set on a `NetNamespace`, but there is no node
 hosting that egress IP, then egress traffic from the namespace will be dropped.
 endif::openshift-origin,openshift-enterprise[]
 
+[[admin-guide-automatic-egress-ip]]
+=== Enabling Automatic Egress IPs
+
+Similar to xref:enabling-static-ips-for-external-project-traffic[Enabling Static
+IPs for External Project Traffic], as a cluster administrator you can assign
+egress IPs to namespaces by setting the `egressIPs` field to the `NetNamespace`
```

Comment:
Are these actually fields, or are they parameter values? They look like parameter values. If they are, I'd s/field/parameter value and s/resource/parameter value throughout this section.